images/dind/node/openshift-generate-node-config.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source /usr/local/bin/openshift-dind-lib.sh
+# Should set OPENSHIFT_NETWORK_PLUGIN
+source /data/network-plugin
+
+function ensure-node-config() {
+  local deployed_config_path="/var/lib/origin/openshift.local.config/node"
```

Comment:
Is it possible multiple nodes are sharing this folder and the last one in is stomping? Trying to figure out what is causing https://github.com/openshift/origin/issues/11274